bgpd/bgpd.c

```diff
@@ -1532,9 +1532,13 @@ struct peer *peer_new(struct bgp *bgp)
 
 	SET_FLAG(peer->sflags, PEER_STATUS_CAPABILITY_OPEN);
 
+	SET_FLAG(peer->flags_invert, PEER_FLAG_CAPABILITY_FQDN);
+
 	if (CHECK_FLAG(bgp->flags, BGP_FLAG_ENFORCE_FIRST_AS))
 		SET_FLAG(peer->flags, PEER_FLAG_ENFORCE_FIRST_AS);
 
+	SET_FLAG(peer->flags, PEER_FLAG_CAPABILITY_FQDN);
```

Comment:
Can you keep this together with `SET_FLAG(peer->flags_invert, PEER_FLAG_CAPABILITY_FQDN);`? Just for readability.

Reply:
done